docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+## Changelog
```

Comment:
I think `changelog` and `description` are almost the same thing. I prefer to keep only `description`.